android/app-example/src/main/java/com/badoo/ribs/example/rib/dialog_example/dialog/RibDialog.kt

```diff
@@ -0,0 +1,20 @@
+package com.badoo.ribs.example.rib.dialog_example.dialog
+
+import com.badoo.ribs.dialog.Dialog
+import com.badoo.ribs.dialog.Dialog.Event.Negative
+import com.badoo.ribs.dialog.Dialog.Event.Positive
+import com.badoo.ribs.example.rib.lorem_ipsum.builder.LoremIpsumBuilder
+
+class RibDialog(
+    loremIpsumBuilder: LoremIpsumBuilder
+) : Dialog<Dialog.Event>({
+        title = "A title if you wish"
+        ribFactory {
```

Comment:
Factory is not serializable, is it fine?

Reply:
I guess this is in reference to passing the dialog around in a Configuration data class. I guess if it's a RIB inside, it already has its whole infrastructure for dynamic updates (input deps or internal feature impl).

Reply:
Will it handle this dialog correctly after restore though?

Reply:
Not this one, but that's the point - Dialogs that host RIBs do not even need to be serialised and passed inside the Configuration.